inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/weighable_layer_without_fq.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
From my point of view this test is quite specific for GNA since weightable layer (ConcatAlignFilter) will be created only in GNA, for other plugins it's just concatenation of the input and a constant, so consider to move it into GNA internal tests, for example 'scale_factors_tests'.
Also I'd mention ConcatAlignFilter in the test name or comment, since it can be not evident what weightable layer is tested here.

Reply:
done